package.json

```diff
@@ -34,7 +34,7 @@
     "migrate:latest": "knex $npm_package_config_knex migrate:latest",
     "migrate:rollback": "knex $npm_package_config_knex migrate:rollback",
     "seed:run": "knex $npm_package_config_knex seed:run",
-    "wait-4-docker": "node ./scripts/_wait4_all.js"
+    "wait-4-docker": "node ./scripts/_wait4_all.mjs"
```

Comment:
what do you think about using transpiled files @lewisdaly ?

Reply:
You mean so that are backwards compatible with older node versions?